server.js

```diff
@@ -1,12 +1,23 @@
 const express = require("express")
 
 const app = express()
+const mongoose = require('mongoose')
+const data ='mongodb+srv://nitinsoni:<Nitin11>@most-searched-web.cc9wsj9.mongodb.net/?retryWrites=true&w=majority'
 
+app.get("/",(req,res)=>{
```

Comment:
The connection to the database should not be made inside a route handler. [important]